lib/rubocop/rspec/config_formatter.rb

```diff
@@ -19,6 +19,7 @@ def dump
 
       def unified_config
         cops.each_with_object(config.dup) do |cop, unified|
+          next if cop.eql?('RSpec/Cop')
```

Comment:
This config formatter grabs the configuration for each cop. It receives a set of keys from YARD under the `RuboCop::Cop::RSpec` namespace. It skips `RSpec/Cop` since it is not a cop and doesn't appear in our config file